client/src/layouts/User/User.js

```diff
@@ -78,10 +78,7 @@ export default function User({ ...rest }) {
     }
   }
   React.useEffect(() => {
-    if (
-      currentUser &&
-      (currentUser.userRole == 'Admin' || currentUser.userRole == 'admin')
-    ) {
+    if (currentUser && currentUser.userRole === 'Admin') {
```

Comment:
why are we keeping role as Capitalized name. keep it small case only.

Reply:
All constant values used in EMS starts with capital letters only.. 'Admin' is one of those ...

Reply:
either all shall be small case or all shall be in upper came case .. please confirm ..

Reply:
Done. Also updated API for same